Cargo.toml

```diff
@@ -2,6 +2,7 @@
 members = [
   "objects",
   "miden-lib",
+  "tx",
```

Comment:
Having a directory named just `tx` feels a bit weird to me. Maybe we still keep it as `miden-tx` even though it breaks consistency a bit?